vllm/model_executor/models/bart.py

```diff
@@ -303,7 +304,7 @@ def __init__(
                              f" and `num_heads`: {num_heads}).")
         self.scaling = self.head_dim**-0.5
 
-        self.qkv_proj = QKVParallelLinear(
+        self.qkv_proj = QCrossKVParallelLinear(
```

Comment:
Was wondering if in the encoder-decode e2e test (e.g. test_bart.py) did you observe any increase in the input token/s or the output token/s with this change?

Reply:
Good question. I ran a very informal test to prove to myself that QCrossKVParallelLinear improved the total runtime of the encoder/decoder example script. I should run an actual benchmark in order compare tokens/s